pbj-core/config/detekt/detekt.yml

```diff
@@ -0,0 +1,807 @@
+build:
+  excludeCorrectable: false
+
+config:
+  validation: true
+  warningsAsErrors: false
+  checkExhaustiveness: false
+  # when writing own rules with new properties, exclude the property path e.g.: ['my_rule_set', '.*>.*>[my_property]']
+  excludes: []
+
+processors:
+  active: true
+  exclude:
+    - 'DetektProgressListener'
+  # - 'KtFileCountProcessor'
+  # - 'PackageCountProcessor'
+  # - 'ClassCountProcessor'
+  # - 'FunctionCountProcessor'
+  # - 'PropertyCountProcessor'
+  # - 'ProjectComplexityProcessor'
+  # - 'ProjectCognitiveComplexityProcessor'
+  # - 'ProjectLLOCProcessor'
+  # - 'ProjectCLOCProcessor'
+  # - 'ProjectLOCProcessor'
+  # - 'ProjectSLOCProcessor'
+  # - 'LicenseHeaderLoaderExtension'
```

Comment:
Minor: could we remove commented-out lines here and in a few places below? And/or if this file comes from somewhere externally, can we add a comment at the top of the file with a reference to the original, similar to how checkstyle configs above mention Palantir?

PS. I'm also unsure how much of this config is relevant to PBJ (all those native code mentions below, js, android, and whatnot) and perhaps most lines could be removed to fall back to defaults? Or maybe nuke this entire file? What will break if we do?

Reply:
These, honestly, I think are useless. They're identical from the consensus node repo, but don't seem to get picked up

Reply:
OK I just removed them.